_techwatch/network.md

```diff
@@ -12,3 +12,9 @@ layout: page
 * auto-gen TOC:
 {:toc}
 
+## Working Documents ##
+*   https://twiki.cern.ch/twiki/bin/view/HEPIX/TechwatchNetwork/WebHome
```

Comment:
@edoardomartelli links in Markdown don't work like this. As you wrote it, it will appear just as a text. To have it turned into a link, you need to use the following syntax:

```html
[Some Text](https://twiki.cern.ch/twiki/bin/view/HEPIX/TechwatchNetwork/WebHome)
```

If you want the anchor text to be the link url, you need:
```html
[https://twiki.cern.ch/twiki/bin/view/HEPIX/TechwatchNetwork/WebHome](https://twiki.cern.ch/twiki/bin/view/HEPIX/TechwatchNetwork/WebHome)
```